blivet/populator/helpers/__init__.py

```diff
@@ -1,4 +1,5 @@
 import inspect as _inspect
+import six as _six
```

Comment:
Yeah, it's probably only relevant for `import *`. At some point I was noticing some junk in the namespaces in blivet from this type of import but, to be honest, I don't remember the exact details.